### PR TITLE
feat: upgrade rkyv

### PR DESCRIPTION
**Pull request by lightsing**

### CODE REVIEW

**Review by lispc — APPROVED** [review]

File: crates/core/src/chunk.rs [hunk 1]
```diff
@@ -31,6 +31,7 @@ impl ChunkInfo {
         let post_state_root = traces.last().expect("at least 1 block needed").root_after();
         let withdraw_root = traces.last().unwrap().withdraw_root();
 
+        cycle_tracker_start!("Keccak::v256");
```
Comment (matthiasgoergens) [inline crates/core/src/chunk.rs:34]: What does this have to do with upgrading rkyv?
